src/structures/game_round.ts

```diff
@@ -140,19 +184,19 @@ export default class GameRound {
      * @returns the number of points as defined by the mode type and correctness of the guess
      */
     checkGuess(guess: string, guessModeType: GuessModeType): number {
+        let pointReward = 0;
         if (guessModeType === GuessModeType.SONG_NAME) {
```

Comment:
Now that the return statements are gone in each of these blocks, can't else it's be used?